ThunderCloud/SplitViewController.swift

```diff
@@ -32,8 +32,8 @@ open class SplitViewController: UISplitViewController {
 		
 		let rightVC = placeholderDetailVCClass.init(nibName: nil, bundle: nil)
 		detailViewController = SplitViewController.navigationController(for: rightVC)
-		
-		super.init(nibName: nil, bundle: nil)
+    
+        super.init(nibName: nil, bundle: nil)
```

Comment:
@BenShutt This is one of the changes between previous PR and this one. We can't use the `init(style:)` init method if we still want the primary VC to be a `UITabBarViewController`